doc/getting_started/installation_mac.rst

```diff
@@ -63,6 +63,52 @@ Install tools to build Zephyr binaries:
    $ rm get-pip.py
    $ pip3 install --user -r scripts/requirements.txt
 
+Install exactly version 3.8.2 of CMake:
+
+* Download :file:`cmake-3.8.2-Darwin-x86_64.dmg` from `CMake downloads`_
+* Open the .dmg file and drag the CMake.app into your Applications folder
+* Add the :file:`/Applications/CMake.app/Contents/bin` to your path:
+
+.. code-block:: console
+
+   $ echo "export PATH=/Applications/CMake.app/Contents/bin:\$PATH" >> $HOME/.bash_profile
```

Comment:
@jukkar had a good point about it being better to modify the PATH in zephyrrc, this ensures we don't mess up the host system.

I'm going to fix this in the Linux instructions ASAP